signer/src/api.rs

```diff
@@ -40,6 +40,34 @@ pub enum MessageParams {
     PaychConstructorParams(fil_actor_paych::ConstructorParams),
     #[serde(with = "paych::UpdateChannelStateParamsAPI")]
     UpdateChannelStateParams(fil_actor_paych::UpdateChannelStateParams),
+    //#[serde(with = "miner::MinerConstructorParamsAPI")]
```

Comment:
perhaps cleanup all the commented out ones?

Reply:
unless it's a TODO...

Reply:
Yeah it is a TODO. It is because there is some unknown types in those parameters that we don't know how to serialize/deserialize in JSON. However no one asked for them specifically.